apps/daimo-mobile/src/i18n/languages/my.ts

```diff
@@ -0,0 +1,897 @@
+import { LanguageDefinition } from "./en";
+
+export const my: LanguageDefinition = {
+  // Common text components
+  shared: {
+    buttonStatus: {
+      success: () => `ပေးပို့မှုအောင်မြင်သည်`,
+      error: () => `မှားယွင်းနေပါသည်`,
+      retry: () => `ထပ်မံပြုလုပ်မည်`,
+      sent: () => `ပေးပို့ပြီးပါပြီ`,
+      request: () => `တောင်းခံမည်`,
+    },
+    buttonAction: {
+      confirm: () => `အတည်ပြုမည်`,
+      cancel: () => `ပယ်ဖျက်မည်`,
+      decline: () => `ငြင်းပယ်မည်`,
+      request: () => `တောင်းခံမည်`,
+      send: () => `ပေးပို့မည်`,
+      accept: () => `လက်ခံမည်`,
+      submit: () => `Submit`,
+      continue: () => `ဆက်လက်ဆောင်ရွက်မည်`,
+    },
+    textPressable: {
+      cancel: () => `ပယ်ဖျက်မည်`,
+    },
+  },
+  // useSendAsync for sending userops
+  sendUserOp: {
+    loadingAccount: () => `Loading account...`,
+    authorizing: () => `Authorizing...`,
+    accepted: () => `လက်ခံလိုက်ပါပြီ`,
+    offline: () => `လုပ်ငန်စဥ်မအောင်မြင်ပါ။ အင်တာနက်လိုင်းကျနေပါသလား?`,
+    error: () => `ပေးပို့ရာတွင်တစ်စုံတစ်ခုအမှားအယွင်းရှိနေပါသည်`,
+  },
+  // -------------------------- SCREENS --------------------------
+  // ------------ HISTORY ------------
+  // HistoryList.tsx
+  historyList: {
+    empty: () => `ပေးပို့ထားမှုများမရှိပါ`,
+    screenHeader: {
+      default: () => `ပေးပို့ထားသောမှတ်တမ်းများ`,
+      other: () => `Between you`,
+    },
+    op: {
+      cancelledLink: () => `Linkကိုပယ်ဖျက်မည်`,
+      pending: () => `ဆောင်ရွက်နေပါသည်`,
+    },
+  },
+  // HistoryOpScreen.tsx
+  historyOp: {
+    shareLinkAgain: () => `Linkကိုထပ်မံမျှဝေမည်`,
+    viewReceipt: () => `ပေးပို့မှုမှတ်တမ်းကိုကြည့်ရှုမည်`,
+    opVerb: {
+      createdLink: () => `Created link`,
+      acceptedLink: () => `Accepted link`,
+      receivedRequest: () => `Received request`,
+      fulfilledRequest: () => `Fulfilled request`,
+      cancelledLink: () => `Cancelled link`,
+      sent: () => `ပေးပို့ပြီးပါပြီ`,
+      received: () => `လက်ခံရရှိသည်`,
+      deposited: () => `Deposited`,
+      depositing: () => `Depositing`,
+      withdrew: () => `Withdrew`,
+      withdrawing: () => `Withdrawing`,
+    },
+    help: {
+      title: () => `ပေးပို့မှုနှင့်ပတ်သက်သောအချက်အလက်များ`,
+      whyNoFees: {
+        firstPara: (chainName: string) =>
+          `ယခုပေးပို့မှုကို Ethereum rollup ဖြစ်သော ${chainName} ပေါ်တွင်ဆောင်ရွက်ပါမည်`,
+        firstPara2Chain: (chainA: string, chainB: string) =>
+          `ယခုပေးပို့မှုသည် ${chainA} နှင့်${chainB} တို့ပေါ်တွင်ဆောင်ရွက်သွားမည်ဖြစ်ပါသည်`,
+        secondPara: () =>
+          `Rollups inherit the strong security guarantees of Ethereum, at lower cost.`,
```

Comment:
the ai might've missed a few spots? nbd

Reply:
not ai, human did it -- they wrote it's better to keep it in english for phrases like this